src/Microsoft.Data.Sqlite/project.json

```diff
@@ -1,6 +1,13 @@
 {
   "version": "1.0.0-*",
-  "description": "SQLite implementation of the System.Data.Common provider model.",
+  "description": "Provides a data provider for SQLite based on the System.Data.Common contract.\r\n\r\nCommonly Used Types:\r\nMicrosoft.Data.Sqlite.SqliteCommand\r\nMicrosoft.Data.Sqlite.SqliteConnection\r\nMicrosoft.Data.Sqlite.SqliteConnectionStringBuilder\r\nMicrosoft.Data.Sqlite.SqliteDataReader\r\nMicrosoft.Data.Sqlite.SqliteException\r\nMicrosoft.Data.Sqlite.SqliteFactory\r\nMicrosoft.Data.Sqlite.SqliteParameter\r\nMicrosoft.Data.Sqlite.SqliteTransaction",
```

Comment:
So you're saying "Provides a ... provider" is weird? :smirk: (I copied it from SqlClient)